### PR TITLE
Hello-Microverse_Project Pull request

### PR DESCRIPTION
**Pull request by Cheptoo21**

### CODE REVIEW

**Comment by Cheptoo21** [comment]
A simple Linters file for html and css

**Review by skyv26 — CHANGES_REQUESTED** [review]
# Hi @Cheptoo21, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## STATUS: REQUIRE CHANGES 🔴 



 ## Highlights
- Correct `GitHub` flow ✅ 

 ## Required Changes ♻️

- [ ] Kindly use the professional *README* template provided by the **Microverse** as suggested by your previous code reviewer in this [PR](https://github.com/Cheptoo21/Hello-Microverse/pull/3) but it seems like you have not implemented the things as mentioned by the previous code reviewer, you can find the `README` template [here](https://github.com/microverseinc/readme-template/blob/master/README.md). Right now as I can see that your project documentation is not looking professional, please adhere to the project requirement and implement the project as per the requirement as you can see down below. 

**NOTE:** *Don't forget to customize the README, except the FAQ section rests of everything is required so don't make any mistake while creating the documentation.*

image

### README comparison

<table>
<tr>
<th align=“middle”>Your README</th>
<th align=“middle”>Professional README</th>
</tr>
<tr>
<td>

image

</td>
<td>

professional-readme

</td>
</tr>
</table>

- [ ] Please configure the `.github/workflows` in your project as a feature properly as mentioned by your previous code reviewer it seems like your previous code reviewer suggested fixing your linters error in this [PR](https://github.com/Cheptoo21/Hello-Microverse/pull/3) but it seems like you messed up with the things by creating a new project with new PR, so please fix the issue which will help you to write industry-standard code and also validate your code to avoid `syntax` errors and for clear code purposes because I can see that you have the `.github/workflows` folder but you don't have the `linters.yml` configuration which you can find it [here](https://github.com/microverseinc/linters-config/tree/master/html-css). Kindly adhere to the project requirement.

image

### ⚠️⚠️ Important Note ⚠️⚠️
**Please don't open the new PR, please make the above changes in this branch and submit this same PR [link](https://github.com/Cheptoo21/Hello-Microverse_Project/pull/2) for getting another review.**


_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag @skyv26 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Cheptoo21 — COMMENTED** [review]
Hey,I already changed my README file

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @Cheptoo21 , 

image

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights
- [ ] All required linters have been setup✔️ 
- [ ] Good commit messages 👍🏾 
- [ ]  Correct GitHub flow ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._
- [ ]  Your project is almost complete but you have a few issues holding you back in the readme. I will list the changes below one by one so that you can fix them all and move one 😄 
- [ ] Delete the broken image link or replace it with your own logo if you have one
logo

- [ ] Delete all instructions & guiding text. These lines begin with the `>` sign. They are meant to give you instructions when filling in the readme.
instr

- [ ] Your project is not deployed, so please delete the live demo section. This will reduce redundancy in your code. 🪧 
demo

- [ ] Pleases avoid the use of **N/A, Comming soon, ...**. If a section is not relevant at the moment, please delete it.  The **Usage section** however is relevant to this project. Please describe how some can use your project. How can they get the project to run on the browser - please describe that in a few words
usage

- [ ] The **FAQs** section is optional. As is the case with all optional sections, they can be deleted if you decide not to fill them in.






## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by wayungi** [comment]
@Cheptoo21, please note that you still have 3 extra reviews to use. Do not give up thinking that you have used up all your reviews. reach me on zoom if you need clarification

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (bobb-Rob) [inline index.html:3]: - [ ] It appears that you have not fixed any issues raised by the previous reviewer, kindly fix them as they will improve your project and make it even better.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
```
Comment (bobb-Rob) [inline index.html:9]: - [x] [OPTIONAL] This pull request does not have any summary/description, please add a brief detail of what was done on this project. 

image

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️

Hi @Cheptoo21, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Cheptoo21** [comment]
Hey,I think you're reviewing the wrong part,I was only told to review the
README file.

Please be considerate.

Thank you

**Comment by ahmetbozaci** [comment]
### Hello @Cheptoo21 
### _This is an additional review as a result of the second opinion request._ 

- The code reviewer @bobb-Rob is right to mention issues again mention in this [review](https://github.com/Cheptoo21/Hello-Microverse_Project/pull/2#pullrequestreview-1454640644). Because you didn't implement the changes in `Linter-branch` you implement the changes in the `main` branch which is not correct. 
- For now, we can approve the project because you implemented the changes. But in your future projects please don't implement the changes in the main branch. It was mention in the [requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md#:~:text=Therefore%20the%20main%20branch%20should%20always%20be%20stable%20code%20that%20actually%20works%20and%20can%20be%20safely%20used.%20Any%20changes%20required%20in%20the%20application%20should%20be%20introduced%20in%20feature%20branches%20and%20approved%20before%20merging%20them%20to%20the%20main%20branch.)
- [ ] Please open all links in the project [requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) and read them clearly. 


**_As this was the only reason your project was not approved it will be approved as soon as possible._**

___


### Cheers and Happy coding! 💻 👏
____
[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by topeogunleye — APPROVED** [review]
### Hi @Cheptoo21, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
 
## [Optional] suggestions:


Your branch name dev does not follow the convention of being named after the feature or action that you implemented. For example, if you are working on Hello Microverse project, you could name your branch `feature/Hello-Microverse`. This makes it easier to track the changes and the purpose of each branch.

- [ ] Next time, and for future Pull Requests, please kindly add a more descriptive PR title and description as seen in the demo below:
pull_request_desc 

- [ ] Kindly ensure your linter are running and the results pass * AKA: Linter should report NO errors.

 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**